### DISCUSSION TITLE
Add the Azure Functions Python 3 template?

### DISCUSSION THREAD

**Discussion by jcbhmr** [devcontainers-community/templates#25] [Ideas]
next on the list after deno (#24) is the azure functions template for python 3. @eitsupi opinions?

![image](https://user-images.githubusercontent.com/61068799/235494213-af504598-9e4b-4221-84cb-66b2e89af5b6.png)

heres a list of other azure functions templates besides the python3 one fyi

![image](https://user-images.githubusercontent.com/61068799/235494082-1f74a0ec-131a-4c53-a174-166bf0e6b4ab.png)

**Comment by eitsupi** [comment]
Thanks for digging to that.

These seem to be templates created by the first party (The Azure Functions Team).
So instead of keeping them here they could create their own repository and maintain it.

For example, the contents of the Dockerfile looks like the following, which seems to be completely managed by them

https://github.com/microsoft/vscode-dev-containers/blob/5a084a93b0736ea86395ac99019a5b72a00b6341/containers/azure-functions-python-3/.devcontainer/Dockerfile

```dockerfile
# Find the Dockerfile at this URL
# https://github.com/Azure/azure-functions-docker/blob/dev/host/4/bullseye/amd64/python/python39/python39-core-tools.Dockerfile
FROM mcr.microsoft.com/azure-functions/python:4-python3.9-core-tools
```

**Reply by jcbhmr** [reply]
good thoughts! 👍 as a curious azure user, id love to open a pr for those to improve the template, but the current template is deprecated... 😢 is there a way you think would be best to reach out and ask about creating a https://github.com/Azure/devcontainer-templates or similar repo? on their https://github.com/Azure/azure-functions-docker/issues docker issue tracker maybe?

**Reply by eitsupi** [reply]
The commit history shows that it was created primarily by @anthonychu.
Perhaps it would be best to mentions @anthonychu in https://github.com/microsoft/vscode-dev-containers?

@anthonychu Hi, if you don't mind, could you tell us if you plan to maintain some Dev Container Templates for Azure Functions in https://github.com/microsoft/vscode-dev-containers?

**Reply by anthonychu** [reply]
Adding @kulkarnisonia16 to help with this.

**Comment by jcbhmr** [comment]
Any updates on this? Should we just skip this for now?
